.github/workflows/build_and_test_workflow.yml

```diff
@@ -422,6 +422,15 @@ jobs:
       - name: Build `${{ matrix.name }}`
         run: yarn ${{ matrix.script }} --release
 
+      - name: Run `${{ matrix.name }}`
+        run: ./artifacts/target/${{ env.ARTIFACT_BUILD_NAME }}/bin/opensearch-dashboards &
+        shell: bash
+
+      - name: Health check 
+        run: |
+          timeout 300 bash -c 'while [[ "$(curl http://localhost:5601/api/status | jq -r '.status.overall.state')" != "green" ]]; do sleep 5; done'
+        shell: bash
+
```

Comment:
Should catch issues like: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5952

Reply:
It seems like this workflow is not being run currently, so can't verify it works. @bandinib-amzn can you help me run it/can you enable it from your end just so we can verify?

Reply:
The linux artifact is used in BWC tests and I don't think OSD will come up "green" without an OS instance to connect to.